causal_automl/notebooks/TutorTask235_Exploratory_analysis_of_FRED_metadata.py

```diff
@@ -0,0 +1,2385 @@
+# ---
```

Comment:
I would move all the code in a lib, so it's easier to maintain / review.
Also pls search the helpers for code, since we want to reuse as much code as possible.

@sonniki I think we have a lot of EDA code in cmamp, that we should maybe we should move to helpers to reuse.

The long-term goal is to automate / reuse as much of the EDA flows as possible.